### PR TITLE
Update adopters.csv

### PR DESCRIPTION
**Pull request by lucliscio**

### CODE REVIEW

**Comment by CoralineAda** [comment]
That's version 1.4. I would recommend using the 2.0 version instead: https://www.contributor-covenant.org/it/version/2/0/code_of_conduct/

File: static/adopters.csv [hunk 1]
```diff
@@ -90,6 +90,7 @@ First Contributions, https://github.com/firstcontributions/first-contributions
 Freedomotic, https://github.com/freedomotic/freedomotic
 friends, https://github.com/JacobEvelyn/friends
 Fukuzatsu, https://gitlab.com/coraline/fukuzatsu/tree/master
+funKounter, https://github.com/lucliscio/fankounter
```
Comment (mheob) [inline static/adopters.csv:115]: Is the name `funKounter` spelled correctly?
I suppose it should be called `fanKounter`.

Comment (lucliscio) [reply static/adopters.csv:115]: yes the correct name is fanKounter.... I apologize for my mistake

**Review by CoralineAda — APPROVED** [review]

File: static/adopters.csv (hunk 1 above)
Comment (Paulina1234567890) [reply static/adopters.csv:115]: Esta mal

**Pull request merged**